### PR TITLE
Fix internal pipeline

### PR DESCRIPTION
**Pull request by catalinaperalta**
Fix problems with integration tests yaml. Make sure we're in the SDK path, fix environment var reference and use gocov.

### CODE REVIEW

File: eng/integration-tests.yml [hunk 1]
```diff
@@ -37,15 +39,18 @@ steps:
...
+      gocov convert cover.out > coverage.json
+      gocov-xml < coverage.json > coverage.xml
+      gocov-html < coverage.json > coverage.html
     displayName: 'Run Integration Tests'
     env:
       ARM_SUBSCRIPTION_ID: $(go-live-azure-subscription-id)
       ARM_CLIENT_ID: $(go-live-eh-azure-client-id)
       ARM_CLIENT_SECRET: $(go-live-eh-azure-client-secret)
-      ARM_TENANT_ID: $(go-live-azure-tenant-id)
+      ARM_TENANT_ID: $(go-live-tenant-id)
```
Comment (jhendrixMSFT) [inline eng/integration-tests.yml:53]: Is this the only one without a `azure-` in its name?

**Review by jhendrixMSFT — APPROVED** [review]

File: eng/integration-tests.yml (hunk 1 above)
Comment (catalinaperalta) [reply eng/integration-tests.yml:53]: Yes :( We could update it to have more consistency, but this will also affect SB.

**Comment by jonasrichard** [comment]
@jhendrixMSFT Hey, in the chanelog.md I refer to a version like v3.3.3 which contains this fix but the tag is missing. It would be good to tag the merge commit, in order that everything will be correct.

**Comment by jhendrixMSFT** [comment]
This fix is in the CI yaml file.  While we can certainly tag it, it doesn't affect the module's content in any way.

**Pull request merged**